docs/misc/changelog.rst

```diff
@@ -42,6 +42,7 @@ Bug Fixes:
   `Issue #430 <https://github.com/hill-a/stable-baselines/issues/430>`_.
 - Fix a bug when calling `logger.configure()` with MPI enabled (@keshaviyengar)
 - set `allow_pickle=True` for numpy>=1.17.0 when loading expert dataset
+- Fix a bug when use VecCheckNan to check env and use numpy ndarray as state.  `Issue #489 <https://github.com/hill-a/stable-baselines/issues/489>`_.
```

Comment:
Please at your username + at the bottom too ;)

Reply:
Thank you very much. It's my first to create a PR and I'm a little excited to do something for such a big project. :)